RUFAS/input_manager.py

```diff
@@ -1943,3 +1989,130 @@ def dump_get_data_logs(self, path: Path) -> None:
         file_name = om.generate_file_name(base_name="InputManager_get_data_log", extension="json")
         file_path = os.path.join(path, file_name)
         om.dict_to_file_json(self.__get_data_logs_pool, file_path)
+
+
+class ElementState(Enum):
+    """
+    An enumeration of the states an input data element can be in during validation. An element cannot
+    be in more than one state at a time.
+
+    Attributes
+    ----------
+    VALID : int
+        The element is valid.
+    INVALID : int
+        The element is invalid and cannot be fixed.
+    FIXED : int
+        The element is invalid initially but has been fixed.
+    """
+
+    VALID = 1
+    INVALID = 2
+    FIXED = 3
```

Comment:
```suggestion
    VALID = "valid"
    INVALID = "invalid"
    FIXED = "fixed"
```
Definitely more redundant, but is clearer.

Reply:
Strings make more intuitive sense to me as well, but for my own edification, is there any benefit in using the existing ints?